src/services/ParticipantProjectSpeciesService.ts

```diff
@@ -146,6 +144,8 @@ const list = async (
 const update = (participantProjectSpecies: ParticipantProjectSpecies): Promise<Response2<UpdateResponse>> => {
   const entity: UpdateRequestPayload = {
     feedback: participantProjectSpecies.feedback,
+    internalComment: participantProjectSpecies.internalComment,
```

Comment:
We also need to update the list function to get the "internalComment" field. It's not showing in the accelerator species view

Reply:
Not suer if it's the search, because it should come from the get, but the internal comment is not showing